packages/dx-react-grid/src/plugins/table-header-row.test.jsx

```diff
@@ -92,15 +93,28 @@ describe('TableHeaderRow', () => {
       </PluginHost>
     ));
 
+    const tableCellProps = defaultDeps.template.tableCell;
+
     expect(isHeadingTableCell)
       .toBeCalledWith(
-        defaultDeps.template.tableCell.tableRow,
-        defaultDeps.template.tableCell.tableColumn,
+        tableCellProps.tableRow,
+        tableCellProps.tableColumn,
       );
-    expect(tree.find(defaultProps.cellComponent).props())
+    expect(tree.find(defaultProps.cellLayoutComponent).props())
```

Comment:
Where are the tests for the Cell component?

Reply:
CellComponent is not created directly by the TableHeaderRow. It is created by TableCellLayout, so all the test cases are there.

Reply:
Do this mean that we are not testing the Cell component anymore?